### PR TITLE
SUP-21596: remove deleted entries from elastic results

### PR DESCRIPTION
**Pull request by amiras89**
This change is [Reviewable](https://reviewable.io/reviews/kaltura/server/9395)

### CODE REVIEW

**Comment by amiras89** [comment]
Disclaimer:
We fetch 10 objects at a time
If one of them is deleted - we will fetch 9 results (while the request said pageSize=10)

File: plugins/search/providers/elastic_search/lib/model/search/kEntrySearch.php [hunk 1]
```diff
@@ -107,6 +107,17 @@ public function getElasticTypeName()
     public function fetchCoreObjectsByIds($ids)
     {
         $entries = entryPeer::retrieveByPKsNoFilter($ids);
+
+        // remove deleted entries from results (in case there is major lag - elastic may return IDs that are deleted in DB)
+		foreach ($entries as $key => $entry)
```
Comment (MosheMaorKaltura) [inline plugins/search/providers/elastic_search/lib/model/search/kEntrySearch.php:112]: Fix indentation.
Verify that if you have related objects such as caption on entry , and it was removed - that the result will not break.

Comment (amiras89) [reply plugins/search/providers/elastic_search/lib/model/search/kEntrySearch.php:112]: Due to complexity and chance of breaking flow - we decided to write a script to fix it.

**Comment by amiras89** [comment]
Due to complexity and chance of breaking backward flow - we decided to write a script to delete entries from elastic manually

**Pull request closed without merging**